### ISSUE TITLE
multipy runtime + dynamo E2E test

### ISSUE THREAD

**Issue by d4l3k** [pytorch/multipy#93]
Would be nice to have a runtime/deploy + dynamo test to ensure they work seamlessly out of the box

**Comment by d4l3k** [comment]
There's still some more work that needs to be done before we can re-enable them with prims support https://github.com/pytorch/multipy/blob/main/multipy/runtime/test_compat.py#L27
